internal/server/files.go

```diff
@@ -0,0 +1,184 @@
+package server
+
+import (
+	"bufio"
+	"encoding/base64"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+
+	"github.com/labstack/echo/v4"
+)
+
+type files struct{}
+
+type Resource struct {
+	RemoteIdentifier string `json:"remoteIdentifier"`
+}
+
+type ValetRequestParams struct {
+	Operation string `json:"operation"`
+	Resources []Resource
+}
+
+type ValetToken struct {
+	Authorization string `json:"authorization"`
+	FileId        string `json:"fileId"`
+}
+
+func (token *ValetToken) GetFilePath() string {
+	// TODO: Check format of fileId (Security)
+	// TODO: Allow custom path in config
+	// TODO: Subfolders for each user (Compatible format with official server)
+	return "/etc/standardfile/database/" + token.FileId
```

Comment:
Do we have the `UserID` somewhere to add one more namespace in the path?

Reply:
The `UserID` could be derived `token.Authorization`.